sycl/unittests/helpers/PiMockPlugin.hpp

```diff
@@ -1054,3 +1055,21 @@ inline pi_result mock_piTearDown(void *PluginParameter) { return PI_SUCCESS; }
 inline pi_result mock_piPluginGetLastError(char **message) {
   return PI_SUCCESS;
 }
+
+// Returns the wall-clock timestamp of host for deviceTime and hostTime
+inline pi_result mock_piGetDeviceAndHostTimer(pi_device device,
+                                              uint64_t *deviceTime,
+                                              uint64_t *hostTime) {
+
```

Comment:
It would be nice to have a unittest which checks that this API is called when it's supposed to(e.g. profiling enabled) and that SYCL API returns corresponding values.